signalsample.txt

```diff
@@ -1,16 +1,79 @@
-BTC
-ETH
+ZEN

```

Comment:
Leave these default, make your own custom list if you want to use it.

Reply:
done, will be reset on next push - thank you!